brave-spancollector-scribe/src/test/java/com/github/kristofa/brave/scribe/CountingScribeSpanCollectorNotificationsHandler.java

```diff
@@ -0,0 +1,53 @@
+package com.github.kristofa.brave.scribe;
+
+import com.twitter.zipkin.gen.Span;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+class CountingScribeSpanCollectorNotificationsHandler implements ScribeSpanCollectorNotificationsHandler {
```

Comment:
something like this is probably better in the test package, as it is an interesting example, but not actually used, and easy to recreate. I'd expect monitoring bindings to be direct, ex to a codahale or otherwise library, as that allows the most relevant features to be used.